bot/exts/holidays/valentines/be_my_valentine.py

```diff
@@ -30,7 +32,7 @@ def load_json() -> dict:
         return loads(p.read_text("utf8"))
 
     @in_month(Month.FEBRUARY)
-    @commands.group(name="lovefest")
+    @commands.command(name="lovefest", help=f"NOTE: This command has been moved to {MOVED_COMMAND}")
     async def lovefest_role(self, ctx: commands.Context) -> None:
```

Comment:
Could you add `, *, _)` to the function signature here, so that if a user runs `.lovefest sub` this command still gets invoked and they are told about the move, rather than getting a bag arg error.

Reply:
Not necessary. 
![image](https://user-images.githubusercontent.com/71233171/144685196-766ae3bb-5a75-49af-818d-e345dab15fee.png)

Reply:
Ah nice, that works